realtime/utils.js

```diff
@@ -408,16 +408,29 @@ function attachAspectSubject(sample, subjectModel, aspectModel) {
     sample.subject ? sample.subject : subjectModel.findOne(subOpts),
   ])
   .then((response) => {
-    let asp = response[0];
-    let sub = response[1];
-    asp = asp.get ? asp.get() : asp;
+    const ASPECT_INDEX = 0;
```

Comment:
it's a tiny thing, but you can move the definition of these two constants up to the top of the module

Reply:
done